src/main/java/org/repodriller/scm/SCM.java

```diff
@@ -68,7 +68,7 @@
 	Commit getCommit(String id);
 	/* TODO A method named getCommitXYZ should return a Commit. */
 	String getCommitFromTag(String tag);
-
+	void getDiffBetweenCommits(String priorCommit, String laterCommit);
```

Comment:
Just checking...Do we consistently use Strings to identify Commits throughout RepoDriller? Should we?

Reply:
I haven't poked around the RepoDriller source, but while using it I've had to pass Strings. Seems to me like its a 'library agnostic' way of identifying specific commits. That's useful because we might tend to use different abstractions of commits (RepoDriller's `Commit`, JGit's `RevCommit` or `ObjectId`).